test_runner/src/test/kotlin/ftl/args/IosArgsTest.kt

```diff
@@ -245,6 +245,7 @@ IosArgs
         - $testIpa2
       scenario-numbers: 
       type: xctest
+      app: 
```

Comment:
`nonDefault` should contain some value

Reply:
@piotradamczyk5 problem here is it will cause a conflict if i provide something here. Will get a validate exception.
`"Cant have xctest and app defined, must be game-loop"` 
WDYT?

Reply:
ok sure, so leave it 👍